bin/welcome.js

```diff
@@ -0,0 +1,10 @@
+#! /usr/env/bin node
+
+const chalk = require( 'chalk' );
+
+console.log( chalk.cyan( '             _                           ' ) );
+console.log( chalk.cyan( '    ___ __ _| |_   _ _ __  ___  ___      ' ) );
+console.log( chalk.cyan( '   / __/ _\` | | | | | \'_ \\/ __|/ _ \\ ' ) );
+console.log( chalk.cyan( '  | (_| (_| | | |_| | |_) \\__ \\ (_) |  ' ) );
+console.log( chalk.cyan( '   \\___\\__,_|_|\\__, | .__/|___/\\___/ ' ) );
+console.log( chalk.cyan( '               |___/|_|                \n' ) );
```

Comment:
actually @DanReyLop I think this is a major blocker. it's _text_ are and we need emoji. I just don't see how we could move forward without proper gyphage